pinot-core/src/main/java/org/apache/pinot/core/util/GeoColumnUtils.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.core.util;
+
+import java.util.List;
+import java.util.Optional;
+import org.apache.pinot.common.request.context.ExpressionContext;
+
+
+public final class GeoColumnUtils {
+
+  private GeoColumnUtils() {
+  }
+
+  public static Optional<String> getColumnFromArgument(ExpressionContext argument) {
```

Comment:
add some comments

also, i think this handles first level func only

Reply:
got it. added comments and also handled multiple level.

Reply:
Thanks, it's possible for other nested functions such as some geometry manipulation.